TCAT/Controllers/RouteDetail+ContentViewController.swift

```diff
@@ -473,11 +474,11 @@ class RouteDetailContentViewController: UIViewController {
         return drawerDisplayController
     }
 
-    required convenience init(coder aDecoder: NSCoder) {
-        guard let route = aDecoder.decodeObject(forKey: "route") as? Route
-            else { fatalError("init(coder:) has not been implemented") }
-
-        self.init(route: route, currentLocation: nil, routeOptionsCell: nil)
-    }
+//    required convenience init(coder aDecoder: NSCoder) {
+//        guard let route = aDecoder.decodeObject(forKey: "route") as? Route
+//            else { fatalError("init(coder:) has not been implemented") }
+//
+//        self.init(route: route, endDestination: , currentLocation: nil, routeOptionsCell: nil)
+//    }
```

Comment:
Looks like this can be deleted unless there's a reason for it to be commented out?